specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-04-02-preview/managedClusters.json

```diff
@@ -3278,6 +3284,10 @@
           "title": "Whether each node is allocated its own public IP.",
           "description": "Some scenarios may require nodes in a node pool to receive their own dedicated public IP addresses. A common scenario is for gaming workloads, where a console needs to make a direct connection to a cloud virtual machine to minimize hops. For more information see [assigning a public IP per node](https://docs.microsoft.com/azure/aks/use-multiple-node-pools#assign-a-public-ip-per-node-for-your-node-pools). The default is false."
         },
+        "enableCustomCATrust": {
+          "type": "boolean",
```

Comment:
Was this reviewed in an API review meeting? I have some questions like: 
- Should this really be an enum? Could there possibly be multiple "modes" to this in the future?
- Should we be expanding CA in the name here? _Generally speaking_ I don't love abbreviations in the API, but they're ok in cases things are very well understood terms (VM, ID, DNS, etc). CA might meet the bar there but I am not sure that it was discussed.
- Should this go into some sort of `Profile`? A `SecurityProfile` or somesuch on AP, rather than just just jamming everything into the top level?

These sorts of things would hopefully have been covered in an official API review internally but I'm not sure we had one where this was explicitly discussed.

Reply:
(from offline discussion)

the bool would be enabled/disabled, so bool should be fine. 

as far as profiles -- maybe something for future discussion? we didn't see a reason to break with the pattern, e.g. this isn't related to some set of features, but I don't love that we perpetually add One More Knob to AP either.